InteractiveHtmlBom/Run.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+echo                                                                                                                                                                                    -
+echo                                  Thankyou For Using Generate InteractiveBom
+echo                                              Powered By Qu1ck
+echo                                Bat Version : Powered By Scarrrr0725
```

Comment:
This is better:

Thank you for using InteractiveHtmlBom
https://github.com/openscopeproject/InteractiveHtmlBom
Bat file by Scarrrr0725

Reply:
ok....sorry for my english